awx/ui_next/src/screens/Template/shared/SurveyList.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useCallback } from 'react';
+import { withI18n } from '@lingui/react';
+
+import useRequest from '@util/useRequest';
+import ContentError from '@components/ContentError';
+import ContentLoading from '@components/ContentLoading';
+import { JobTemplatesAPI } from '@api';
+import SurveyListItem from './SurveyListItem';
+
+function SurveyList({ template }) {
+  const {
+    result: questions,
+    error: contentError,
+    isLoading,
+    request: fetchSurvey,
+  } = useRequest(
+    useCallback(async () => {
+      const {
+        data: { spec },
+      } = await JobTemplatesAPI.readSurvey(template.id);
+
+      return spec.map((s, index) => ({ ...s, id: index }));
```

Comment:
Survey items do not have an id on them for some reason.  I am putting an id on there here so that we can sort them and delete them in upcoming PR and we can satisfy Reacts need for a unique key for array items.